include/crm/common/results.h

```diff
@@ -107,6 +107,7 @@ enum pcmk_rc_e {
     /* When adding new values, use consecutively lower numbers, update the array
      * in lib/common/results.c, and test with crm_error.
      */
+    pcmk_rc_no_input            = -1027,
     pcmk_rc_no_output           = -1026,
```

Comment:
I love how it looks like these were planned together

Reply:
If anyone asks, I'll say this was the plan from the beginning.